WcaOnRails/app/views/server_status/index.html.erb

```diff
@@ -32,67 +32,52 @@
     </div>
   </div>
   <% (I18n.available_locales - [:en]).each do |locale| %>
-    <% missing = @status_locales[locale][:missing] %>
-    <% outdated = @status_locales[locale][:outdated] %>
-    <% all_empty = (missing.empty? && outdated.empty?) %>
-    <% class_heading, class_panel, class_missing, class_outdated, title = css_classes(missing.empty?, outdated.empty?) %>
+    <%
+      bad_keys_by_type = @bad_keys_by_type_by_locale[locale]
+      bad_keys = bad_keys_by_type.values.flatten
```

Comment:
We can use `all_bad_keys` to not override this below (in the block). I'm not strong about this, so do whatever seems appropriate for you.